runtime/frequency/src/lib.rs

```diff
@@ -26,6 +26,9 @@ use sp_runtime::{
 	DispatchError,
 };
 
+pub mod development_genesis;
+pub mod genesis_config_presets;
```

Comment:
Should these be feature flagged to `frequency-local` and `frequency-local`? They shouldn't be in mainnet or testnet right?

Reply:
I’ve been going back and forth on this. The API suggests that you can retrieve a list of all presets, which implies we should have more than just one. I also didn’t notice their example using flags. In any case, I don’t see it as a big deal to keep them. 

How do you feel about feature flagging them once we get all of them in as a final task?

Reply:
Hmm... I don't like the idea of the development code being available in our mainnet wasm. That breaks with our rule of separation.

I'm open to being convinced, but I think we should go ahead and add the feature flags before merging.

I would be fine with merging into a feature development branch then a PR to do the flagging and a final PR to merge it all to main.

Reply:
>Hmm... I don't like the idea of the development code being available in our mainnet wasm. That breaks with our rule of separation.

I agree with that. Less code means less issues. I am curious on the reasoning behind Polkadot not having the same concern.

>I would be fine with merging into a feature development branch then a PR to do the flagging and a final PR to merge it all to main.

That sounds good too me. I'll add to the acceptance criteria of the parent ticket feature flagging the presets.

Reply:
Here’s the link to the new branch that includes moving the Frequency and Paseo Genesis: https://github.com/frequency-chain/frequency/pull/2155